DayNightQ10.Rmd

```diff
@@ -69,3 +69,30 @@ plot(x,y)
 cat("The Q10 Value is", calc_q10(x,y))
 ```
 
+## Ts Percentage
+```{r Ts Percentage, message = FALSE, warning=FALSE}
+main_results <- list()
+for(rhds in rh_dataset_names){
+  ds_data <- csr_dataset(rhds)
+  dsd <- ds_data$data
+  column_names <- names(dsd)
+  temperature_columns <- column_names[grep("^CSR_T(AIR|[0-9]+)", column_names)]
```

Comment:
This is fine, but I think it would be more straightforward to just store the column numbers in `temperature_columns`.  Then all those `results$columns == tcol` instances below just become `tcol`.

Reply:
That may cause my depth column to break though since the name of the column is needed to obtain the depth

Reply:
Let's deal with this one later, then.